contracts/Liquidator/Liquidator.sol

```diff
@@ -351,8 +351,9 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
 
     function _reduceReservesInternal() internal {
         uint256 pendingReedemLength_ = pendingRedeem.length;
+        uint256 chunk = pendingReedemLength_ >= 10 ? 10 : pendingReedemLength_;
```

Comment:
1. I would extract this number to a constant at least
2. I said 10 as an example. It would be great if we can estimate the gas spent at each iteration, to confirm 10 is good enough